src/frontend/src/icp/components/convert/HowToConvertEthereumInfo.svelte

```diff
@@ -23,14 +23,17 @@
 	import { formatToken } from '$lib/utils/format.utils';
 	import { replaceOisyPlaceholders, replacePlaceholders } from '$lib/utils/i18n.utils';
 
-	export let formCancelAction: 'back' | 'close' = 'back';
+	interface Props {
+		formCancelAction: 'back' | 'close';
```

Comment:
maybe this is necessary? or not?

```suggestion
		formCancelAction?: 'back' | 'close';
```

Reply:
yep, it indeed should be optional! thanks, fixed